version.txt

```diff
@@ -0,0 +1 @@
+0.0.4
```

Comment:
Should be 0.0.3 since that's the current release, no?

Reply:
No, I am thinking that the version.txt holds the version to be released. (I.E the one we are making prereleases on now)

So if we would choose to produce artifacts on our master commits they would be versioned 0.0.4-buildnumber because they are pre-releases of version 0.0.4 (and 0.0.3-buildnumber does not make any sense since version 0.0.3 is already released)

And this is how we do in mira and I think comes from cloud

But If we want to have it the otherway that is fine by me as well :)

Reply:
I personally prefer to keep the version master being the latest release, and instead tag the rolling builds with like `20181004-<buildnumber>` or something similar if needed. You don't know after releasing a version what the next version will be so it doesn't make sense to me that you'd bump it prematurely. :)